tests/Cms/Pages/PageRenderTest.php

```diff
@@ -0,0 +1,401 @@
+<?php
+
+namespace Kirby\Cms;
+
+use Kirby\Cache\Value;
+use Kirby\Exception\NotFoundException;
+use Kirby\Filesystem\Dir;
+use Kirby\TestCase;
+
+/**
+ * @covers \Kirby\Cms\Page::cacheId
```

Comment:
`Todo:` This one doesn't make that much sense for me. If we want to improve the separation what tests do what, this method should be covered in the PageCacheTest class (if needed with a new test), but not by PageRenderClass.

Reply:
`cacheId` is only ever called and used by `render()`.

Reply:
Currently yes, but we should test it then separately in `PageCacheTest`. For me, understanding our tests will be harder if we have a method like `::cacheId()` but its test coverage is not in `PageCacheTest` (the main class testing the caching), but some other class.

Reply:
The only thing that `PageCacheTest` still tests is the `isCacheable` method. Should we rather move those back in to `PageRenderTest` then? Because I think it would be even worse to start adding additional render tests to `PageCacheTest`.

Reply:
If you prefer that, I'd see no downside.